rpc/common/moc_common_security.proto

```diff
@@ -87,3 +89,69 @@ message Scope {
 	ProviderType providerType = 3;
 	string resource = 4;
 }
+
+message KeyCommon {
```

Comment:
wouldn't this break cloudagent? @raghavendra-nataraj

Reply:
I think this wouldn't since the field numbers are constant. However @jirenugo , I think we should not remove the cloudagent struct all together. I was only looking to abstract struct to common key proto like 

message PrivateKeyWrappingInfo {
	string WrappingKeyName = 1;
	bytes WrappingKeyPublic = 2 [(sensitive) = true];
	KeyWrappingAlgorithm WrappingAlgorithm = 3;
}

message SignVerifyParams {
	JSONWebKeySignatureAlgorithm  algorithm  = 1;
	string signature = 2  [(sensitive) = true];
}

Reply:
We should still have the base Key, request, response and methods functions in the cloudagent since they give flexibility in changing the signature if needed

Reply:
> I think this wouldn't since the field numbers are constant

yeah, just moving wouldn't have, but we also renamed key to keycommon